### PR TITLE
fix: improve warning for untranspiled modules

### PR DESCRIPTION
**Pull request by TheAlexLichter**
## Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)

## Description

The improved error message should give users a better idea what to do :thinking: 

image

### CODE REVIEW

File: packages/server/src/middleware/error.js [hunk 1]
```diff
@@ -5,8 +5,11 @@ import consola from 'consola'
 import Youch from '@nuxtjs/youch'
 
 export default ({ resources, options }) => function errorMiddleware(err, req, res, next) {
-  // ensure statusCode, message and name fields
+  if (/unexpected token/gi.test(err.message)) {
```
Comment (pi0) [inline packages/server/src/middleware/error.js]: This error is _originating_ from webpack. So would be better to handle in friendly-errors plugin.

Comment (Atinux) [reply packages/server/src/middleware/error.js]: Here? https://github.com/nuxt/nuxt.js/blob/dev/packages/webpack/src/plugins/warnfix.js

Comment (Atinux) [reply packages/server/src/middleware/error.js]: I will also recommend to add an url to the documentation and avoid `!` that feels frustrating.

Comment (pi0) [reply packages/server/src/middleware/error.js]: I mean add a similar transformer like this for `unexpectedToken.js`:

https://github.com/nuxt/friendly-errors-webpack-plugin/blob/master/src/transformers/moduleNotFound.js

And add it to [`defaultTransformers`](https://github.com/nuxt/friendly-errors-webpack-plugin/blob/master/src/friendly-errors-plugin.js#L12)

Comment (manniL) [reply packages/server/src/middleware/error.js]: Will do ☺️

Comment (clarkdo) [reply packages/server/src/middleware/error.js]: @manniL Can we close this now ?

Comment (manniL) [reply packages/server/src/middleware/error.js]: @clarkdo Feel free to close it, open an issue on https://github.com/nuxt/friendly-errors-webpack-plugin/ and assign me ☺️

**Pull request closed without merging**